lib/fs.js

```diff
@@ -1664,6 +1664,8 @@ fs.appendFileSync = function(path, data, options) {
 function FSWatcher() {
   EventEmitter.call(this);
 
+  this._initialized = false;
```

Comment:
Can you add a comment here mentioning that this should be kept in sync with the `initialized` field of the C++ wrap? (Arguably better if this is an accessor property on the prototype of `FSWatcher`, that way there is no need to set this field in `start` and `close`)